slices/python3.yaml

```diff
@@ -0,0 +1,25 @@
+package: python3
+
+essential:
+  - python3_copyright
+
+slices:
```

Comment:
Should we modify or add a slice to link python3.12 to python3 and possibly python?

Reply:
A link from python to python3.12 is not created by the postinst script of python3 package, so I didn't include that in this slice. Besides, linking python3 to a specific python3.x version will cause conflicts when there are multiple python3.x packages in the same release (e.g. python3.10 and python3.11 in 22.04).

Reply:
I also think we should have the `python3` symlink, as that might be the most useful thing about this SDF.

It is true that it isn't set by this pkg. The symlink is created by `python3-minimal` which is a dependency that is missing from this PR.

Reply:
The latest change artificially creates the python3 symlink, but I don't think that's the right way to do it. The symlink is created by python3-minimal which is a dependency that should be included in this PR.

Reply:
Done.